base/shared/src/main/scala/scalaz/typeclass/Ord.scala

```diff
@@ -0,0 +1,23 @@
+package scalaz
+package typeclass
+
+sealed abstract class Ordering
```

Comment:
probably should `extends Product with Serializable` to avoid the LUB sadness.

```scala
scala> List(LT, GT)
res0: List[Product with Serializable with Ordering] = List(LT, GT)
```

although I'm not sure if that's really going to be a problem with this specific one.